src/tools/metadata-to-ipfs.ts

```diff
@@ -0,0 +1,41 @@
+import fs from "fs";
+// @ts-ignore
```

Comment:
Why?

Reply:
Yeah unfortunalty pinata-sdk doesn't have exported types not in this package not in any community packages :(